src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+const Big = require('big.js');
+
+export const operate = (numberOne, numberTwo, operation) => {
+  numberOne = new Big(numberOne);
+  numberTwo = new Big(numberTwo);
+
+  if(operation === '+'){
+    return numberOne.plus(numberTwo)
+  }
+
+  if(operation === '-'){
+    return numberOne.minus(numberTwo);
+  }
+
+  if(operation === '/'){
+    return numberOne.div(numberTwo);
+  }
```

Comment:
Don't forget to handle division by ZERO on next milestone 😃

Reply:
Thank you for pointing this out!